docs/en/guides/build-device/image-building.md

```diff
@@ -41,9 +45,9 @@ Now, you can list your keys with:
 
     snapcraft list-keys
 
-### 2. Upload the key to the store
+### 2. Register the key with the store
 
-Next, you have to upload it to the store, effectively linking it to your account. During this step, you will be asked to select an existing key and login with your store account credentials.
+Next, you have to register the key in the store, effectively linking it to your account. During this step, you will be asked to select an existing key and login with your store account credentials.
```

Comment:
To be consistent with the title: s/register the key in/register the key with/

Reply:
will do